### PR TITLE
Move to Jekyll

### PR DESCRIPTION
**Pull request by paked**
This PR moves the website over to Jekyll. I've done this as discussed on the Gitter.

Now that we are using a static site generator, it should be easier to write blog posts, tutorials and other things.

Also, everything can now be written in Markdown!

@EtienneBruines please review/merge

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: Engo
+email: harrison@theshoebridges.com
+description: > # this means to ignore newlines until "baseurl:"
+    An open-source 2D game engine written in Go following the Entity-Component-System paradigm.
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "https://engo.io" # the base hostname & protocol for your site
+github_username: EngoEngone
```
Comment (otraore) [inline _config.yml]: Spelt wrong it's EngoEngine

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: Engo
+email: harrison@theshoebridges.com
+description: > # this means to ignore newlines until "baseurl:"
```
Comment (otraore) [inline _config.yml:3]: The comment is not needed it was just an example

**Comment by paked** [comment]
@otraore Fixed!

**Comment by EtienneBruines** [comment]
LGTM. I'm merging to see how it looks (I can always revert/fix if it's broken)

**Comment by paked** [comment]
@EtienneBruines feedback?

edit: nevermind, stupid github lol

**Pull request merged**